### ISSUE TITLE
Download file from editor

### ISSUE THREAD

**Issue by sumanthratna** [play-with-docker/play-with-docker#446]
it'd be nice to be able to download files (like PNGs generated by scripts) from the editor window

**Comment by ptavaressilva** [comment]
That would be a useful feature

**Comment by marcosnils** [comment]
You can currently download files through scp

**Comment by ptavaressilva** [comment]
> You can currently download files through scp

Yes, but I understand the playground's motto is "Don’t let app complexity get in the way of opportunity". Therefore simplicity of use should be a priority. This feature would allow learners to continue exercises where they left off.

**Comment by marcosnils** [comment]
True! Unfortunately, I don't have time to implement this ATM. Contributions are welcome if someone's willing to tackle this one.
